pyproject.toml

```diff
@@ -33,9 +33,15 @@ version = "0.0.0"
 
 [project.optional-dependencies]
 server = [
-  "home-assistant-chip-core==2024.4.1",
+  "aiohttp==3.9.5",
+  "aiorun==2023.7.2",
+  "async-timeout==4.0.3",
+  "coloredlogs==15.0.1",
   "cryptography==42.0.5",
+  "dacite==1.8.1",
```

Comment:
Where is dacite used?

Reply:
Hm, yeah it seems it is no longer used. Dropping it.